src/RaisedButton/RaisedButton.js

```diff
@@ -111,69 +111,68 @@ class RaisedButton extends React.Component {
 
   static propTypes = {
     /**
-     * Override the background color. Always takes precedence unless the button is disabled.
+     * This value will override the default background color for the button,
```

Comment:
Can probably just begin this with 'Override the default value' for consistency with other components. The rest of the description makes sense, and is good additional detail. :+1: